pykokkos/interface/views.py

```diff
@@ -704,6 +706,53 @@ def from_cupy(array) -> ViewType:
 
     return from_numpy(np_array, memory_space, layout)
 
+def is_array(array) -> bool:
+    """
+    Check if an object conforms to enough numpy array standards to be treated as an array
+    Using the function from_cupy() as a standard for bare minimum needed to
+    to convert any array into a numpy array
+
+    :param array: the array of unknown type
+    :returns: a true/false if object is an array
+    """
+    
+    required_attr = ["dtype", "data", "shape", "flags"]
```

Comment:
probably should be global on the module level

Reply:
Do we use it elsewhere? I basically stole this from the from_cupy() function and just rephrased a bit.

Reply:
If a constant, no need to keep it a local var. I prefer we extract.